### PR TITLE
Fixes#2997: Added validator in report bug prompt

### LINKED ISSUE #2997: Report a bug prompt is accepting empty request .
**Describe the bug**
while reporting a bug  , empty request are being accepted .

**To Reproduce**
1. Go to simulator.
2. Click on 'report a bug prompt '.
3.  Submit it with empty fields.
4. See error.

**Expected behavior**
should not accept empty value and should give a warning .

**Screenshots**

ezgif com-gif-maker

**Are you working on this issue? (Yes/No)**
**yes**

### PR DESCRIPTION
**Pull request by aashimawadhwa**
Fixes #2997 

#### Describe the changes you have made in this PR -
Added text validator in report a bug prompt

### Screenshots of the changes (If any) -

Screenshot 2022-03-11 16:58:31

### CODE REVIEW

File: app/views/simulator/edit.html.erb [hunk 1]
```diff
@@ -714,6 +714,15 @@
   <% end %>
 </script>
 
+<!-- report an issue validator -->
+<script>
+  function validator() {
+    if (document.getElementById("issuetext").value.length == 0) {
+      alert("issue cannot be empty")
```
Comment (vedant-jain03) [inline app/views/simulator/edit.html.erb:721]: Please change the prompt alert message.
`issue cannot be empty` -> `Description cannot be empty! `

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]

File: app/views/simulator/edit.html.erb (hunk 1 above)
Comment (aashimawadhwa) [reply app/views/simulator/edit.html.erb]: > Please change the prompt alert message. `issue cannot be empty` -> `Description cannot be empty! `

done!

**Comment by vedant-jain03** [comment]
@pavanjoshi914 Any feedback for this?

File: app/views/simulator/edit.html.erb [hunk 2]
```diff
@@ -714,6 +714,15 @@
   <% end %>
 </script>
 
+<!-- report an issue validator -->
+<script>
+  function validator() {
+    if (document.getElementById("issuetext").value.length == 0) {
+      alert("Description cannot be empty!")
```
Comment (tachyons) [inline app/views/simulator/edit.html.erb:721]: We are trying to remove all existing alerts from the simulator, can we use a inline message or a modal ?

Comment (vedant-jain03) [reply app/views/simulator/edit.html.erb:721]: I guess an inline message would be good somewhat like we have in the login form.

Comment (aashimawadhwa) [reply app/views/simulator/edit.html.erb:721]: Okay , I will change the prompt to inline message .

**Pull request closed without merging**